docs/rep/controller/openapi.yaml

```diff
@@ -0,0 +1,298 @@
+openapi: 3.0.2
+info:
+  title: Rik
+  version: 0.1.0
+paths:
+  /api/v0/workloads.list:
+    get:
+      tags:
+        - Workloads
+      description: List all workload
+      parameters:
+        - required: false
+          schema:
+            title: limit
+            minimum: 1
+            type: integer
+          name: limit
+          in: query
+      responses:
+        '200':
+          description: OK
```

Comment:
Every time you return a list of items (workload, instances, tenants, etc), you want to return an array of items like e.g.:

```yaml
        workloads:
          type: array
          items:
            $ref: '#/components/schemas/Workload'
```

And you only define a schema for your item, not for your list of items.